From 2ce48cb185da2d9042d31c6a4a5f20ba3bfd4b24 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Tue, 1 Sep 2015 13:27:30 +0800
Subject: [PATCH 0384/5242] MLK-11480-01 ARM: dts: add ssi<->wm8962 sound card
 support for imx6sx-sdb board

commit  7182ece0ebda5dc94f67d90fde9af5cbb1ed1d86 from
https://source.codeaurora.org/external/imx/linux-imx.git

add ssi<->wm8962 sound card support for imx6sx-sdb board

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sdb.dtsi |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dtsi b/arch/arm/boot/dts/imx6sx-sdb.dtsi
index 3b3036d..9a80eb0 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dtsi
+++ b/arch/arm/boot/dts/imx6sx-sdb.dtsi
@@ -147,7 +147,7 @@
 	sound {
 		compatible = "fsl,imx6sx-sdb-wm8962", "fsl,imx-audio-wm8962";
 		model = "wm8962-audio";
-		ssi-controller = <&ssi2>;
+		cpu-dai = <&ssi2>;
 		audio-codec = <&codec>;
 		audio-routing =
 			"Headphone Jack", "HPOUTL",
@@ -158,6 +158,7 @@
 			"IN3R", "AMIC";
 		mux-int-port = <2>;
 		mux-ext-port = <6>;
+		hp-det-gpios = <&gpio1 17 1>;
 	};
 
 	panel {
@@ -272,6 +273,7 @@
 		PLLVDD-supply = <&vgen4_reg>;
 		SPKVDD1-supply = <&reg_psu_5v>;
 		SPKVDD2-supply = <&reg_psu_5v>;
+		amic-mono;
 	};
 };
 
-- 
1.7.9.5

